### PR TITLE
chore: Containerize the app using Docker

### PR DESCRIPTION
**Pull request by carlqt**
https://github.com/kumojin/kumojin.github.io/issues/4

This PR adds a dockerfile and updates README to make use of containers

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+WORKDIR /app
+COPY Gemfile Gemfile
+COPY Gemfile.lock Gemfile.lock
+RUN gem install bundler -v 2.1.4 && bundle install
+COPY . .
+
+# Add a script to be executed every time the container starts.
+# COPY entrypoint.sh /usr/bin/
+# RUN chmod +x /usr/bin/entrypoint.sh
+# ENTRYPOINT ["entrypoint.sh"]
```
Comment (jmaitrehenry) [inline Dockerfile:13]: Can you remove these lines please?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.7.4
+RUN apt-get update
```
Comment (jmaitrehenry) [inline Dockerfile:2]: Actually, apt-get update will juste download packe list from repo without installing anything.
Can you remove it?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+RUN mkdir /app
+WORKDIR /app
+COPY Gemfile Gemfile
+COPY Gemfile.lock Gemfile.lock
```
Comment (jmaitrehenry) [inline Dockerfile:6]:
```suggestion
COPY Gemfile Gemfile.lock ./
```

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+RUN gem install bundler -v 2.1.4 && bundle install
```
Comment (jmaitrehenry) [inline Dockerfile:7]: It's should not be necessary to install bundler as it's already pre-installed in the base ruby image

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.7.4
+RUN apt-get update
+RUN mkdir /app
```
Comment (jmaitrehenry) [inline Dockerfile:3]: You don't need to create it manually, the `WORKDIR` directive will create it for you

**Review by jmaitrehenry — CHANGES_REQUESTED** [review]
Thanks for your PR, it's a good start!

**Comment by carlqt** [comment]
Thanks for the review @jmaitrehenry.

Pushed the changes based on your reviews.

**Review by jmaitrehenry — APPROVED** [review]
LGTM

**Pull request merged**